src/cli/src/main/java/org/locationtech/geogig/cli/Console.java

```diff
@@ -70,15 +73,20 @@ public Console(InputStream in, OutputStream out) {
      * <p>
      * {@code true} will only be returned if the console's output is not redirected to a file or
      * piped, this console's output stream is "stdout", and the JVM has been invoked from a terminal
-     * that supports ANSI codes.
+     * that supports ANSI codes. Setting ansi.enabled as a config option will cause this to return
+     * true.
      * <p>
      * If {@link #disableAnsi()} has been called, returns {@code false} immediately.
      * 
      * @return {@code true} if ANSI terminal color codes are supported by the console, {@code false}
      *         otherwise.
      */
     public boolean isAnsiSupported() {
-        return ansiEnabled && ansiSupported;
+        if (forceAnsi) {
+            return true;
+        } else {
```

Comment:
a small code style thing: the else clause is redundant. it could jut be ```if(forceAnsi) {return true}; return ansiEnabled && ansiSupported``` or simply ```return forceAnsi || ansiEnabled && ansiSupported```